README.md

```diff
@@ -26,3 +26,24 @@ Within the "Glyphs" directory, each individual glyph set has a sub-directory und
 - Optionally, any examples useful for making a glyph's usage clear.
 
 Glyphs are specified relative to a standard canvas of size 0.5 x 0.5 inches, based on the SBOL Visual 1.0 "Promoter" and "CDS" symbols.
+
+## Submitting changes to the specification
+
+Before writing text for a non-trivial change, please have it approved by discussion as an Issue and/or SEP.
```

Comment:
I would say: "Any text for a non-trivial change should be approved by discussion as an Issue and/or SEP"

I don't see any reason to push a particular ordering: I have found for my own workflow that it works best to start by making the branch and drafting the change, then building the SEP from that.